pinot-druid/src/test/java/org/apache/pinot/druid/data/readers/DruidSegmentRecordReaderTest.java

```diff
@@ -0,0 +1,141 @@
+package org.apache.pinot.druid.data.readers;
+
+import com.google.common.base.Preconditions;
+import java.io.File;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.apache.pinot.common.data.Schema;
+import org.apache.pinot.core.data.GenericRow;
+import org.apache.pinot.core.data.recordtransformer.CompositeTransformer;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class DruidSegmentRecordReaderTest {
+  private static int wikipedia_size = 39244;
```

Comment:
`private static final int`? 

Also, Java convention for `static final` variable name is all capital. `e.g. WIKIPEDIA_SIZE`

Please make the change for all variables defined here.